packages/datalake-data-export/src/utils/helperUtils.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable functional/no-let */
+/* eslint-disable functional/immutable-data */
+import { randomUUID } from "crypto";
+import { format } from "date-fns";
+import { ExportedCollection } from "../config/models/models.js";
+
+/**
+ * Get the path where to store the ndjson file
+ * - data / ...
+ *   - [collection] /
+ *      - yyyyMMdd /
+ *        - yyyyMMdd_HHmmss_[random_uuid].njson
+ *        - yyyyMMdd_HHmmss_[random_uuid].njson
+ *        - yyyyMMdd_HHmmss_[random_uuid].njson
+ * - count / ...
+ */
+export function getNdjsonBucketKey(
```

Comment:
There is also a test for these helpers method that you did not include in the porting: https://github.com/pagopa/interop-be-reports/blob/1.0.x/jobs/datalake-data-export/src/utils/__tests__/helpers.utils.test.ts

Reply:
Good catch totally miss testing on utils!